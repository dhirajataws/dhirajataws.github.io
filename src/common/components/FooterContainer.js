import styled from 'styled-components';
export const FooterContainer = styled.div`
display:flex;
justify-content:stretch;
position:absolute;
width:100%;
top:75%;
height:25%;
background-color:black;
opacity:0.5;
z-index:2;
`