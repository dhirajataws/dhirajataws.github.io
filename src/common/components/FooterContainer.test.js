import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { FooterContainer } from './FooterContainer';

test('FooterContainer components ', () => {
    const tree = renderer.create(<FooterContainer />).toJSON()
    expect(tree).toMatchSnapshot()
    expect(tree).toHaveStyleRule('display', 'flex')
    expect(tree).toHaveStyleRule('justify-content', 'stretch')
    expect(tree).toHaveStyleRule('position', 'absolute')
    expect(tree).toHaveStyleRule('width', '100%')
    expect(tree).toHaveStyleRule('top', '75%')
    expect(tree).toHaveStyleRule('height', '25%')
    expect(tree).toHaveStyleRule('background-color', 'black')
    expect(tree).toHaveStyleRule('opacity', '0.5')
    expect(tree).toHaveStyleRule('z-index', '2')
})