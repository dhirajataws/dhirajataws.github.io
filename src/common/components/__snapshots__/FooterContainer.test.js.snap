// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterContainer components  1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: stretch;
  -webkit-justify-content: stretch;
  -ms-flex-pack: stretch;
  justify-content: stretch;
  position: absolute;
  width: 100%;
  top: 75%;
  height: 25%;
  background-color: black;
  opacity: 0.5;
  z-index: 2;
}

<div
  className="c0"
/>
`;
